basis/common-exceptions/src/main/java/com/alex_koveckiy/common/exceptions/configuration/ModuleConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.alex_koveckiy.common.exceptions.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Created by alex on 01.03.17.
+ */
+
+@Configuration
+@ComponentScan(basePackages = "com.alex_koveckiy.common.exceptions")
+public class ModuleConfig {
```

Comment:
@Alexey007gold Why you need configurations for this module? There is no spring components in this module.